addons/im_livechat/static/src/legacy/public_livechat.js

```diff
@@ -221,6 +221,15 @@ var LivechatButton = Widget.extend({
                 this._renderMessages();
                 return;
             }
+            case 'mail.message/insert': {
+                const message = this._messages.find(message => message._id === payload.id);
+                if (!message) {
+                    return;
+                }
+                message._body = utils.Markup(payload.body);
```

Comment:
@seb-odoo same ci/security reason as original PR, need to fix conflict at each step of fw-port